apps/labs/posts/code-generator-and-nebari.md

```diff
@@ -0,0 +1,154 @@
+---
+title: 'Bridging Data Science Tools with Code Generator and Nebari'
+published: September 20, 2023
+authors: [aryan-gupta]
+description: 'A summary of my contributions to the Code-Generator Project and PyTorch-Ignite ecosystem in the past few months as Quansight Labs intern and my learnings in the process.'
+category: [PyData ecosystem, Machine Learning, OSS Experience]
+hero:
```

Comment:
You also need to set a `featuredImage` because it's a required field, syntax:

```
featuredImage:
  src: /posts/code-generator-and-nebari/hero.png
  alt: 'The Image of the Code-Generator app'
```